### PR TITLE
add placehlders if no app access

### PR DESCRIPTION
**Pull request by nidhigarg-bmw**

### CODE REVIEW

File: cx-portal/src/components/shared/frame/UserList/index.tsx [hunk 1]
```diff
@@ -131,7 +131,10 @@ export const UserList = ({
           },
           {
             field: 'details',
-            headerName: t('global.field.details'),
+            headerName:
+              t(sectionTitle) === 'Identity Management'
```
Comment (oyo) [inline cx-portal/src/components/shared/frame/UserList/index.tsx:135]: never check for a translation value
1. doesn't work for other languages than english
2. will break when the text changes

**Review by oyo — CHANGES_REQUESTED** [review]
translation dependency and merge conflict

File: cx-portal/src/components/shared/frame/UserList/index.tsx (hunk 1 above)
Comment (nidhigarg-bmw) [reply cx-portal/src/components/shared/frame/UserList/index.tsx]: @oyo Made the changes. Please review again

**Review by oyo — APPROVED** [review]
looks good now

**Review by oyo — APPROVED** [review]

**Pull request merged**